server/mailer/RegistrationMailer.js

```diff
@@ -0,0 +1,26 @@
+import { renderFile } from 'ejs'
+import path from 'path'
+import BaseMailer from './BaseMailer'
+
+export default class RegistrationMailer extends BaseMailer {
+  constructor(user) {
+    super({ to: [process.env.ADMIN_EMAIL], from: process.env.EMAIL_SENDER })
+    this.subject = 'New Registered User'
+    this.user = user
+    this.redirectUrl = process.env.HOME_URL
+  }
+
+  get body() {
+    const registrationEmailTemplate = path.join(
+      __dirname,
+      'mailer',
+      'templates',
+      'registrationTemplate.ejs'
+    )
+
+    return renderFile(registrationEmailTemplate, {
+      ...user,
+      redirect: this.redirectUrl,
```

Comment:
this is probably fine, but if we were to have a `redirect` attribute on a user we'd be overriding it here and it also might cause confusion if `redirect` isn't an attribute of a user (which it isn't) but we're using it as one here. Might be worth not de-structuring and just passing `{ user, redirect: process.env.HOME_URL }` 🤷‍♂️